src/gui/torrentmodel.cpp

```diff
@@ -55,10 +56,33 @@ static QIcon getErrorIcon();
 
 static bool isDarkTheme();
 
+static bool isTextVisible(const QModelIndex& idx, QObject *widget, const BitTorrent::TorrentHandle *torrent)
+{
+    if(!widget) return false;
+
+    TransferListWidget *p = dynamic_cast<TransferListWidget *>(widget);
+    if(!p) return false;
```

Comment:
Shouldn't we return `true` here and in the default branch below? The rest of the code is written with the opposite intent: to specify explicitly where do we need a tooltip, while here we see the the tooltip is need unless we can proof the opposite.